src/dynamo0p3_builtins.py

```diff
@@ -160,7 +160,7 @@ def __str__(self):
     def gen_code(self, parent):
         from f2pygen import AssignGen
         # In this case we're multiplying each element of a field by the
-        # supplied scalar value
+        # supplied scalar value.
```

Comment:
Please add "Modified I. Kavcic Met Office " at the appropriate place as you have done for other files.